router/batchrouter/asyncdestinationmanager/eloqua/utils.go

```diff
@@ -0,0 +1,195 @@
+package eloqua
+
+import (
+	"bufio"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/google/uuid"
+	"github.com/rudderlabs/rudder-server/jobsdb"
+	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/common"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+	"golang.org/x/exp/slices"
+)
+
+func checkEventType(file *os.File) (string, string, error) {
+	file.Seek(0, 0)
```

Comment:
we are doing seek many times, possible to do once and gather all the information